firmware/emonTx34/emonTx34_CM/emonTx34_CM.ino

```diff
@@ -52,7 +52,7 @@ copy the following into emonhub.conf:
 #define RFM69_JEELIB_NATIVE 2
 #define RFM69_LOW_POWER_LABS 3
 
-#define RadioFormat RFM69_LOW_POWER_LABS
+#define RadioFormat RFM69_JEELIB_CLASSIC
```

Comment:
The default RadioFormat should be `RFM69_LOW_POWER_LABS`, please could you change this

Reply:
oh sorry. I forgot those subsequent commits would get added to the PR instead of just staying on my fork.